lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,559 @@
+---
+name: Parameters
+field_name_base_url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: modify
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be ChargeItem
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: credit
+    url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: status
+    required: 'Yes'
+    type: code
+    description: The current state of the charge.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "resource": {
+              "status": "unknown"
+            }
+          }
+        ]
+      }
```

Comment:
I think for this operation, we only need a chargeItem and transanctionId parameter example since those are the only parameters we support on this call.

We can give a singular example of a valid ChargeItem object and leave it at that. @alexbeckner1 does this sound good to you?

Reply:
Just to expand on this a bit, if we specify that a parameter must be of type ChargeItem, that already implies all of the subfields included and then we could just add notes for any extra validations beyond the normal ones for modify.

Reply:
Keifer and I discussed this offline and decided to keep the chargeItem parameter subfield separated in the documentation, but only include the subfield structure in the example. Similar to what was done for Create on: https://github.com/cerner/fhir.cerner.com/pull/778

Reply:
updated throughout the file https://github.com/cerner/fhir.cerner.com/pull/780/commits/851312df1b650432b1463c74d6aa86cfe9d025ec#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R31-R33